pyriemann/utils/tangentspace.py

```diff
@@ -99,11 +99,8 @@ def exp_map_logeuclid(X, Cref):
 
     The projection of a matrix :math:`\mathbf{X}` from tangent space
     to SPD/HPD manifold with log-Euclidean exponential map
-    according to a reference SPD/HPD matrix :math:`\mathbf{C}_\text{ref}` is:
-
-    .. math::
-        \mathbf{X}_\text{original} =
-        \exp(\mathbf{X} + \log(\mathbf{C}_\text{ref}))
+    according to a reference SPD/HPD matrix :math:`\mathbf{C}_\text{ref}` as
+    described in [1]_.
```

Comment:
We should cite Eq.(3.4) of Arsigny2007, _Geometric means in a novel vector space structure on symmetric positive-definite matrices_
```suggestion
    described in Eq.(3.4) of [1]_:

    .. math::
        \mathbf{X}_\text{original} = \exp \left( \log(\mathbf{C}_\text{ref})
        + D_{\mathbf{C}_\text{ref}} \log . \mathbf{X} \right)

    where :math:`[D_{\mathbf{C}_\text{ref}} \log . \mathbf{X}` indicates the
    directional derivative of the matrix logarithm at
    :math:`\mathbf{C}_\text{ref}` along :math:`X`.
```

Reply:
Yes, that would be the proper original citation, but the paper does not show how to compute the directional derivative. Not sure if that helps.